ddtrace/tracer/sampler.go

```diff
@@ -147,3 +154,225 @@ func (ps *prioritySampler) apply(spn *span) {
 	}
 	spn.SetTag(keySamplingPriorityRate, rate)
 }
+
+type rulesSampler struct {
+	rules   []SamplingRule
+	rate    float64
+	limiter *rate.Limiter
+	// "effective rate" calculations
```

Comment:
Can we add an empty line above this one? `mu` should be grouped with what it guards.

Reply:
Done